### DISCUSSION TITLE
How to install the extension manually?

### DISCUSSION THREAD

**Discussion by HubKing** [zerodytrash/Simple-YouTube-Age-Restriction-Bypass#234] [Q&A] [answered]
> However, the extension can still be installed manually.

I tried to download the ZIP file and went to FireFox's Add-ons Manager, and chose install from file, but I got this:

![image](https://github.com/zerodytrash/Simple-YouTube-Age-Restriction-Bypass/assets/10900489/c35e0025-f25c-49b7-9f90-f5227dcc4e75)

If it can still be installed, could you add the instruction there?

**Comment by drunkwinter** [comment, marked as answer]
Yes, the readme is not very clear in that regard.

Firefox restricts you from installing any add-on that is not available on their add-ons website. You can technically install it via the `about:debugging` page, but it only remains installed until you restart Firefox. Practically, doing this won't make any sense unless you are a developer and want to test stuff. So it should probably read "**cannot** be installed manually". However I think there are still Firefox forks which do let you install unverified add-ons.
